tests/model_forms/tests.py

```diff
@@ -2553,6 +2556,15 @@ def test_custom_error_messages(self):
             str(errors['name2']),
             '<ul class="errorlist"><li>Model custom error message.</li></ul>'
         )
+        errors = form2.errors
+        self.assertHTMLEqual(
+            str(errors['name1']),
+            '<ul class="errorlist"><li>new message for form2</li></ul>'
+        )
+        self.assertHTMLEqual(
+            str(errors['name2']),
+            '<ul class="errorlist"><li>Model custom error message.</li></ul>'
+        )
```

Comment:
Test that I added to the ticket was just an example. I think we should add a separete test to the `tests/forms_tests/tests/test_forms.py`, e.g.
```python
    def test_error_messages_deep_copy(self):
        class CustomCharField(CharField):
            def __init__(self, **kwargs):
                kwargs['error_messages'] = {'invalid': 'Form custom error message.'}
                super().__init__(**kwargs)

        field = CustomCharField()
        field2 = copy.deepcopy(field)
        self.assertIsInstance(field2, CustomCharField)
        self.assertIsNot(field2.error_messages, field.error_messages)
```